api/java/org/openyolo/api/CredentialClient.java

```diff
@@ -129,8 +144,17 @@ public Intent getHintRetrieveIntent(final HintRetrieveRequest request) {
      * @param callback Handler for the result of the credential request.
      */
     public void retrieve(
-            final CredentialRetrieveRequest request,
+            CredentialRetrieveRequest request,
             final RetrieveCallback callback) {
+
+        // If requireUserMediation was not explicitly set, fallback to isAutoSignInDisabled.
```

Comment:
None come to mind. Agreed, going with OR for until there's a reason to do otherwise.